### DISCUSSION TITLE
Curso FullStack > Executando as migrations

### DISCUSSION THREAD

**Discussion by DaviGadelhaQA** [balta-io/forum#654] [Tira Dúvidas] [answered]
Quando coloco o comando dotnet ef migrations add v1, aparece a seguinte mensagem: Especifique qual arquivo de projeto será usado, pois este 'C:\Developer\Dima\Dima.Api' contém mais de um arquivo de projeto. Qual a forma correta para especificar?
![Captura de tela 2024-08-07 160728](https://github.com/user-attachments/assets/47fcacbb-863c-4f72-8f86-a35814376eff)

**Comment by DaviGadelhaQA** [comment, marked as answer]
Já consegui resolver esse problema

**Reply by andrebaltieri** [reply]
@DaviGadelhaDev posta aqui como resolveu, pode ajudar mais pessoas!
